src/screens/AuthenticatorList/AuthenticatorListScreen.tsx

```diff
@@ -84,12 +84,14 @@ class AuthenticatorListScreen extends Component<Props> {
   renderItem = ({ item }: { item: Authenticator }) => (
     <TouchableOpacity style={styles.authenticatorWrapper} onPress={() => this.navigateToOptions(item.id)}>
       <View style={styles.authenticatorLeftColumn}>
-        <Text style={styles.name}>{item.name}</Text>
+        <Text numberOfLines={1} ellipsizeMode="tail" style={styles.name}>
```

Comment:
we are using it so many times that it would be nice to make a component from it, ElipisisText for example